### ISSUE TITLE
Slow / Incomplete rendering of TableHeader tooltips in Flatlaf 3.1.1 (Windows 11)

### ISSUE THREAD

**Issue by schliepRon** [JFormDesigner/FlatLaf#672]
When moving the cursor on a TableHeader the tooltip rendering appears to be slower than before on windows 11.

The borders are rendered first, without the content, the content follows slightly later.

Screenshots:
![image](https://user-images.githubusercontent.com/52451273/235098577-7439f797-8f87-4e08-9bf6-1cf556185cf0.png)

**Comment by DevCharly** [comment]
Hmm, never seen this. 😕 

> When moving the cursor on a TableHeader ...

Does it work normal on other components?

Can you reproduce it with following test app?

~~~java
import javax.swing.JButton;
import javax.swing.JFrame;
import javax.swing.JPanel;
import javax.swing.SwingUtilities;
import com.formdev.flatlaf.FlatLightLaf;

public class TooltipTest {
    public static void main( String[] args ) {
        SwingUtilities.invokeLater( () -> {
            FlatLightLaf.setup();

            JButton button = new JButton( "hover to show tooltip" );
            button.setToolTipText( "Hello" );

            JPanel panel = new JPanel();
            panel.add( button );

            JFrame frame = new JFrame( "TooltipTest" );
            frame.setDefaultCloseOperation( JFrame.EXIT_ON_CLOSE );
            frame.setBounds( 50, 50, 400, 200 );
            frame.getContentPane().add( panel );
            frame.setVisible( true );
        } );
    }
}
~~~

**Comment by DevCharly** [comment]
@schliepRon have you tried the above test app?

Anybody else seen this issue?

**Comment by schliepRon** [comment]
@DevCharly I couldn`t reproduce it that way.

Also seems to be client specific. Not all my colleagues got that problem. 
But I hadn`t much time to really analyze the problem.

I`ll try to analyze it asap.

**Comment by mellmann** [comment]
I am observing this problem in my project. I modified the example from above by @DevCharly.

In this example, the location and the content of the tooltip are updated continuously as the mouse is moved over the JPanel (which is a functionality offered by the JComponent). 

Without `FlatLightLaf.setup();`, the tooltip is updated seemlessly, but with Flatlaf there is a considerable delay in redrawing and flickering as reported by @schliepRon.

```java
import javax.swing.JButton;
import javax.swing.JFrame;
import javax.swing.JPanel;
import javax.swing.SwingUtilities;
import com.formdev.flatlaf.FlatLightLaf;
import java.awt.Dimension;
import java.awt.Point;
import java.awt.event.MouseEvent;

public class TooltipTest {
    public static void main( String[] args ) {
        SwingUtilities.invokeLater( () -> {
            FlatLightLaf.setup();

            JPanel panel = new JPanel() {
                @Override
                public String getToolTipText(MouseEvent e) {
                    return String.format("%d;%d", e.getX(), e.getY());
                }
                @Override
                public Point getToolTipLocation(MouseEvent e)
                {
                  Point p = e.getPoint();
                  p.y += 15;
                  return p;
                }
            };
            
            panel.setToolTipText( "" );

            JFrame frame = new JFrame( "TooltipTest" );
            frame.setDefaultCloseOperation( JFrame.EXIT_ON_CLOSE );
            frame.setBounds( 50, 50, 400, 200 );
            frame.getContentPane().add( panel );
            frame.setVisible( true );
        } );
    }
}
```

**Comment by DevCharly** [comment]
@mellmann thanks for the test case.

There is an open issue here: #832

**Comment by DevCharly** [comment]
Fixed in latest `3.5-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots
